client/app/account/billing/main/history/downloadBills/billing-main-history-downloadBills.controller.js

```diff
@@ -0,0 +1,108 @@
+angular.module('Billing').controller('BillingMainHistoryDonwloadBillsCtrl', class BillingMainHistoryDonwloadBillsCtrl {
+  constructor($q, $translate, $uibModalInstance, Alerter, OvhApiMe) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.$uibModalInstance = $uibModalInstance;
+    this.Alerter = Alerter;
+    this.OvhApiMe = OvhApiMe;
+  }
+
+  $onInit() {
+    this.archiveName = `${this.$translate.instant('billing_main_bills_tab')}_${moment().format('YYYY_MM_DD_hh_mm_ss')}`;
+    this.billsCount = null;
+    this.apiV7Request = this.OvhApiMe.Bill().v7().query().sort('date', 'DESC');
+  }
+
+  hasError() {
+    return !!this.error || !!this.apiError;
+  }
+
+  isDateFromBeforeDateTo() {
+    this.error = null;
+    if (!this.searchTo) {
+      return true;
+    }
+
+    const searchFrom = moment(this.searchFrom, 'YYYY-MM-DD');
+    const searchTo = moment(this.searchTo, 'YYYY-MM-DD');
+
+    if (moment(searchFrom).isAfter(searchTo)) {
+      this.error = this.$translate.instant('billing_main_bill_download_date_from_after_date_to');
```

Comment:
```suggestion
      this.messageError = this.$translate.instant('billing_main_bill_download_date_from_after_date_to');
```

Reply:
This looks like it should be handled by the view instead of the controller

Reply:
```suggestion
      this.errorMessage = this.$translate.instant('billing_main_bill_download_date_from_after_date_to');
```